ci-operator/step-registry/hypershift/mce/install/disconnected/OWNERS

```diff
@@ -0,0 +1,19 @@
+approvers:
+- csrwng
+- enxebre
+- sjenning
+- imain
+- LiangquanLi930
+- heliubj18
+- rokej
+- dtthuynh
+options: {}
+reviewers:
+- csrwng
+- enxebre
+- sjenning
+- imain
+- LiangquanLi930
+- heliubj18
```

Comment:
I think @bryan-cox and me already have deep knowledge on disconnected area, could you add us to the OWNERS maybe?